src/ol/format/IIIFInfo.js

```diff
@@ -0,0 +1,368 @@
+/**
+ * @module ol/format/IIIFInfo
+ */
+
+import {assert} from '../asserts.js';
+
+
+/**
+ * @typedef {Object} PreferredOptions
+ * @property {string} [format] Preferred image format. Will be used if the image information
+ * indicates support for that format.
+ * @property {string} [quality] IIIF image qualitiy.  Will be used if the image information
+ * indicates support for that quality.
+ */
+
+/**
+ * @typedef {Object} SupportedFeatures
+ * @property {Array<string>} [supports] Supported IIIF image size and region
+ * calculation features.
+ * @property {Array<string>} [formats] Supported image formats.
+ * @property {Array<string>} [qualities] Supported IIIF image qualities.
+ */
+
+/**
+ * Supported image formats, qualities and supported region / size calculation features
+ * for different image API versions and compliance levels
+ * @const
+ * @type {Object<string, Object<string, SupportedFeatures>}
+ */
+const IIIF_PROFILE_VALUES = {
+  version1: {
+    level0: {
```

Comment:
These unquoted properties are subject to renaming by a compiler or minifier.  Since elsewhere you look up these properties using the `Version` enum below, you should construct this object using the same enum values.

In addition, if the `level0` etc values are parsed from remote data (it looks like they are), they should be quoted when used as properties to avoid renaming.

E.g.
```js
const IIF_PROFILE_VALUES = {};
IIF_PROFILE_VALUES[Versions.VERSION1] = {
  'level0': {} // etc
};
```

Reply:
Done.